api/users/serializers.py

```diff
@@ -1,6 +1,37 @@
 from rest_framework import serializers as ser
 from api.base.serializers import JSONAPISerializer, LinksField, Link
 from website.models import User
+from rest_framework.fields import SkipField, get_attribute
+
+
+class CharFieldWithReadDefault(ser.CharField):
+
+    def get_attribute(self, instance):
+        """
+        Overwrite the error message to return a blank value is if there is no existing value.
+        This allows the display of keys that do not exist in the DB (gitHub on a new OSF account for example.)
+        """
+        try:
```

Comment:
This is almost identical to the parent class's implementation. Instead of copying the code, you could call the parent class's `get_attribute` and handle the `KeyError` and `AttributeError`.